app/pages/event13/Event13CheckYourAnswersPage.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.event13
+
+import models.enumeration.EventType.Event13
+import pages.{CheckAnswersPage, Waypoint, Waypoints}
+import play.api.mvc.Call
+
+case class Event13CheckYourAnswersPage() extends CheckAnswersPage {
+  override val urlFragment: String =
+    s"event-${Event13.toString}-check-answers"
+
+  override def route(waypoints: Waypoints): Call = {
+    controllers.event13.routes.Event13CheckYourAnswersController.onPageLoad
+  }
+
+  override def toString: String = "CheckYourAnswersPage"
+}
+
+object Event13CheckYourAnswersPage {
+
+  def waypointFromString(s: String): Option[Waypoint] = {
+    val pattern = """event-13-check-answers"""
+    s match {
+      case x if x == pattern =>
```

Comment:
More concise to say case "event-13-check-answers" =>

Reply:
Resoved

Reply:
I think you can just say (even simpler):-
    s match {
      case "event-13-check-answers" => Some(Event13CheckYourAnswersPage().waypoint)
      case _ => None
    }

Reply:
Done